From e0c1a86bf88b568a7afe8ebaea1b9f84afb892c2 Mon Sep 17 00:00:00 2001
From: Eli Schwartz <eschwartz@archlinux.org>
Date: Wed, 4 May 2022 23:52:31 -0400
Subject: [PATCH] meson: get the project version from the version-gen script

This matches autotools and ensures that the version number is actually
reliable.
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/meson.build
+++ b/meson.build
@@ -1,5 +1,5 @@
 project('util-linux', 'c',
-        version : '2.37',
+        version : run_command('tools/git-version-gen', check: true).stdout(),
         license : 'GPLv2+')
 
 pkgconfig = import('pkgconfig')
